src/visualizations.jl

```diff
@@ -37,7 +37,8 @@ See also [`abmvideo`](@ref) and [`abmexploration`](@ref).
 * `offset = nothing` : If not `nothing`, it must be a function taking as an input an
   agent and outputting an offset position tuple to be added to the agent's position
   (which matters only if there is overlap).
-* `scatterkwargs = ()` : Additional keyword arguments propagated to the `scatter!` call.
+* `agentsplotkwargs = ()` : Additional keyword arguments propagated to the `agentsplot!` 
+  call.
```

Comment:
Here we refer to an `agentsplot!` call, but this is not explained in the docstring. A user that has not familiarized themselves with the developer internals of the plotting won't know what this means. We could say something like "Additional keyword arguments propagated to the function that plots the agents (typically `scatter!`).".

Reply:
You're right. Fixed that.